src/main/scala/scalismo/common/Scalar.scala

```diff
@@ -0,0 +1,283 @@
+package scalismo.common
```

Comment:
The file header is missing. 
Are parts of it copied from spire? If yes, we should mention it